core/config/schema.go

```diff
@@ -136,15 +136,15 @@ type ConfigSchema struct {
 	OCRContractConfirmations                   uint                          `env:"OCR_CONTRACT_CONFIRMATIONS"`
 	OCRContractPollInterval                    time.Duration                 `env:"OCR_CONTRACT_POLL_INTERVAL" default:"1m"`
 	OCRContractSubscribeInterval               time.Duration                 `env:"OCR_CONTRACT_SUBSCRIBE_INTERVAL" default:"2m"`
-	OCRContractTransmitterTransmitTimeout      time.Duration                 `env:"OCR_CONTRACT_TRANSMITTER_TRANSMIT_TIMEOUT" default:"10s"`
+	OCRContractTransmitterTransmitTimeout      time.Duration                 `env:"OCR_CONTRACT_TRANSMITTER_TRANSMIT_TIMEOUT" default:"2s"`
```

Comment:
Alright, thanks for the clarification! I will change the implementation.
I think it does not need to be changed for other chains, just BSC.

Reply:
@samsondav I did it the way you suggested, but looks like we cannot remove the default tags from schema.go, unless we change the type from `time.Duration` to `*time.Duration`. This is because if no default is specified, it tries to create a zero value with `reflect.New(item.Type).Interface()` and this gives us `*time.Duration`, which makes sense since we need to indicate a missing value. So, I need a piece of advice here how to handle this better: to change type and usages, or do something different? Thanks!